ingest/src/main/resources/log4j2.xml

```diff
@@ -28,25 +28,34 @@
                 <SizeBasedTriggeringPolicy size="10 MB"/>
             </Policies>
         </RollingFile>
+        <File name="FileAppender" fileName="/var/log/hydra/hydra-ingest.log">
+            <PatternLayout pattern="%d{yyyy-MM-dd HH:mm:ss} %-5p (%C:%L) - %m%n" />
+        </File>
```

Comment:
Should we add rolling policy like here: https://github.com/pluralsight/hydra-kstreams/blob/master/server/src/main/resources/logback.xml#L32

Reply:
Sure. I added a size-based rolling policy of 3GB. Is it fine?